src/pages/documentation/current-sdk-versions.md

```diff
@@ -132,3 +132,19 @@ If you are currently using the ACP-prefixed Flutter libraries, please see [the g
 | [EdgeConsent](../consent-for-edge-network/index.md) | [![pub package](https://img.shields.io/pub/v/flutter_aepedgeconsent.svg)](https://pub.dartlang.org/packages/flutter_aepedgeconsent) |
 | [Edge Bridge](../adobe-analytics/migrate-to-edge-network.md) | [![pub package](https://img.shields.io/pub/v/flutter_aepedgebridge.svg)](https://pub.dartlang.org/packages/flutter_aepedgebridge) |
 | [UserProfile](../profile/index.md) | [![pub package](https://img.shields.io/pub/v/flutter_aepuserprofile.svg)](https://pub.dartlang.org/packages/flutter_aepuserprofile) |
+
+## Roku
+
+<div style="height:20px; width:120px;"></div>
+
+<div style="height:30px; width:140px;">
+<a href="https://github.com/adobe/aepsdk-roku/releases">
+<img src="https://img.shields.io/github/v/release/adobe/aepsdk-roku.svg?label=AEP%20Roku%20SDK&logo=github"/>
+</a>
+</div>
+
+The Adobe Experience Platform SDK for Roku supports Roku OS 11.0 or later. The project is open sourced in GitHub. For more information, refer to the documentation below in the [aepsdk-roku](https://github.com/adobe/aepsdk-roku) repository:
+
+* [Getting Started](https://github.com/adobe/aepsdk-roku/blob/main/Documentation/getting-started.md)
+* [API Reference](https://github.com/adobe/aepsdk-roku/blob/main/Documentation/api-reference.md)
+* [Sample App](https://github.com/adobe/aepsdk-roku/blob/main/sample/simple-videoplayer-channel/README.md)
```

Comment:
for this page can we keep just the link to the repo similar with the other extensions, reading:
The project is open sourced in GitHub. For more information, refer to the [aepsdk-roku](https://github.com/adobe/aepsdk-roku) repository.